docs/static/hour-of-code/index.html

```diff
@@ -0,0 +1,247 @@
+                            </a>
+                        </div>
+                        <div class="content-column one-third">
+                            <a href="https://arcade.makecode.com/#tutorial:/tutorials/chase-the-pizza" target="_blank">
+                                <img src="./chase-the-pizza.png" alt="Chase the Pizza" />
+                                <div class='action-button'>Chase the Pizza</div>
+                            </a>
+                        </div>
+                        <div class="content-column one-third">
+                            <a href="https://arcade.makecode.com/#tutorial:/tutorials/lemon-leak" target="_blank">
+                                <img src="./lemon-leak.png" alt="Lemon Leak" />
+                                <div class='action-button'>Lemon Leak</div>
+                            </a>
+                        </div>
+                        <div class="content-column one-third">
+                            <a href="https://arcade.makecode.com/#tutorial:/tutorials/galga" target="_blank">
+                                <img src="./galga.png" alt="Galga" />
+                                <div class='action-button'>Galga</div>
+                            </a>
+                        </div>
+                        <div class="content-column one-third">
+                            <a href="https://arcade.makecode.com/#tutorial:/tutorials/collect-the-clovers" target="_blank">
+                                <img src="./clovers.png" alt="Catch the Clovers" />
+                                <div class='action-button'>Catch the Clovers</div>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
+            <section class='container explore'>
+                <div class="content-block">
+                    <h2 class="display-font">Explore More MakeCode</h2>
+                    <div class='text-content-block'>Whether it's gaming, hacking, creating, or learning, 
+                        you can use MakeCode to bring your project to life. Explore everything MakeCode Arcade has to 
+                        offer, or check out our other platforms such as MakeCode Micro:Bit and MakeCode Minecraft.</div>
+                    <div class="content-row">
+                        <div class="content-column one-third">
+                            <a href="https://arcade.makecode.com/" target="_blank">
+                                <img src="./arcade.png" alt="MakeCode Arcade" />
+                                <div class='action-button'>MakeCode Arcade</div>
+                            </a>
+                        </div>
+                        <div class="content-column one-third">
+                            <a href="https://makecode.microbit.org/" target="_blank">
+                                <img src="./micro-bit.png" alt="MakeCode Micro:bit" />
```

Comment:
```suggestion
                                <img src="./micro-bit.png" alt="MakeCode micro:bit" />
```

my understanding is micro:bit is always styled as lowercase ~

Reply:
Fixed to micro:bit!